Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -236,8 +236,32 @@ def self.check_http_content(url, name, user_agents: [:default])
       details[:content_length] == secure_details[:content_length]
     file_match = details[:file_hash] == secure_details[:file_hash]
 
-    return if !etag_match && !content_length_match && !file_match
-    "The URL #{url} could use HTTPS rather than HTTP"
+    if etag_match || content_length_match || file_match
+      return "The URL #{url} should use HTTPS rather than HTTP"
```

Comment:
Use `Formatter.url(url)` for the URLs.

Reply:
`#{url}` is used this way throughout this whole source file, so I think moving to `Formatter.url(url)` would better be done in a separate PR addressing it for all lines.
Unless I missing something and above is a special case, but since it is an existing line, I'm not sure.

Reply:
Yep, a separate PR is probable a better idea.